AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Internal/PredictionsAction.swift

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+public enum PredictionsAction {
```

Comment:
Does this need to be public? (having it live under `Internal` made me wonder)

Reply:
That's actually a holdover from v1 that doesn't need to be there anymore. Thanks!
[f981b90](https://github.com/aws-amplify/amplify-swift/pull/2901/commits/f981b90b75aa9036a23924abbc031eacfbad697f)